stable/traefik/templates/storeconfig-job.yaml

```diff
@@ -18,6 +18,16 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
       restartPolicy: Never
+      affinity:
```

Comment:
This should be in the config (similar to existing values file `affinity`, somewhere understandable, such as under `kvprovider`, or `kvprovider.storeAcme` etc). 

If any existing keys are changed it would be a MAJOR bump. See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning